guides/rhai_expressions_cheat_sheet.cheatmd

```diff
@@ -0,0 +1,54 @@
+# Rhai expressions cheatsheet
+
+We are grouping into this cheatsheet the most frequent manipulations that can be done through Rhai, for convenience.
+
+## Cheatsheet
+{: .col-2}
+
+### Lists
+
+#### Filtering a list
+
+```rust
+[1, 2, 3].filter(|value| value % 2 == 0)
+=> [2]
+```
+
+#### Finding an element inside a list
+
+```rust
+repl> ["wow", "check"].index_of(|value| value == "check")
```

Comment:
This finds the index, not the element.
I would keep this by changing the title and adding another cheat describing how to get an element by a property in a list of maps.

`nodelist[nodelist.index_of(|x| x.nodeId == 1)]`
where `nodelist = [#{nodeId: 1}, ....]`

Reply:
index_of() returns -1, if the element searched by '==' is not available. And this means, that the 'cheat' will return the last element of the list/array, which is NOT the expected result.

Reply:
Luckily `.find()` by @fabriziosestito got merged in the last release